compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/PlatformWindowContext.desktop.kt

```diff
@@ -72,14 +73,31 @@ internal class PlatformWindowContext {
     fun convertWindowToLocalPosition(container: Component, positionInWindow: Offset) =
         positionInWindow - offsetInWindow(container).toOffset(container.density)
 
+    /**
+     * If the [component]'s location on screen is undefined, returns [Offset.Unspecified]; otherwise
+     * passes its location on screen to [block] and returns its return value.
+     */
+    private inline fun withLocationOnScreenOrUnspecified(
+        component: Component,
+        block: (locationOnScreen: Offset) -> Offset
+    ): Offset {
+        if (!component.isShowing) return Offset.Unspecified
+        (SwingUtilities.getWindowAncestor(component) as? Frame)?.let { frame ->
+            if (frame.state == Frame.ICONIFIED) return Offset.Unspecified
+        }
```

Comment:
Nitpick: `let` makes it worse readable here
```suggestion
        val frame = SwingUtilities.getWindowAncestor(component) as? Frame
        if (frame?.state == Frame.ICONIFIED) return Offset.Unspecified
```

Reply:
`frame?.state` boxes unnecessarily.
Changed to
```
val frame = SwingUtilities.getWindowAncestor(component) as? Frame
if ((frame != null) && (frame.state == Frame.ICONIFIED)) return Offset.Unspecified
```